## /bin/sh -e simulate dpatch for prepend_chlog.py
## Build
## Makefile_hostcc.dpatch by Kilian Krause <kilian@debian.org>
## DP: Remove gcc hardcoding (Closes: #316802)

Index: asterisk-1.2.27/Makefile
===================================================================
--- asterisk-1.2.27.orig/Makefile	2008-03-19 11:33:20.000000000 +0100
+++ asterisk-1.2.27/Makefile	2008-03-19 11:33:39.000000000 +0100
@@ -19,8 +19,8 @@
 # CROSS_COMPILE=/opt/montavista/pro/devkit/arm/xscale_be/bin/xscale_be-
 # CROSS_COMPILE_BIN=/opt/montavista/pro/devkit/arm/xscale_be/bin/
 # CROSS_COMPILE_TARGET=/opt/montavista/pro/devkit/arm/xscale_be/target
-CC=$(CROSS_COMPILE)gcc
-HOST_CC=gcc
+#CC=$(CROSS_COMPILE)gcc
+HOST_CC=$(CC)
 # CROSS_ARCH=Linux
 # CROSS_PROC=arm
 # SUB_PROC=xscale # or maverick
